app/templates/macros/badges.html

```
@@ -47,3 +47,29 @@
+{% macro obsazeni_badges(event, verbose=false) %}
+  {%- set mf = event.mandatory_filled_spots -%}
+  {%- set mt = event.mandatory_total_spots -%}
+  {%- set of = event.optional_filled_spots -%}
+  {%- set ot = event.optional_total_spots -%}
+  {%- set mand_cls = 'bg-success' if mt > 0 and mf >= mt else 'bg-danger' -%}
+  {%- set opt_cls  = 'bg-success' if ot > 0 and of >= ot else 'bg-warning text-black' -%}
+  {%- if verbose -%}
+    {%- if mt > 0 %}<span class="badge {{ mand_cls }}">{{ mf }}/{{ mt }} nutných pozic obsazeno</span>{% endif -%}
+    {%- if ot > 0 %} <span class="badge {{ opt_cls }}">{{ of }}/{{ ot }} volitelných pozic obsazeno</span>{% endif -%}
+  {%- else -%}
+    {%- if mt > 0 %}<span class="badge {{ mand_cls }}">{{ mf }}/{{ mt }}</span>{% endif -%}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)